src/styles/reset.scss

```
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
  margin: 0;
}
```
As Andrii said, it's a good practice to reset margins/paddings in * selector.